pnglibconf.h

```diff
@@ -0,0 +1,216 @@
+/* libpng 1.6.18 STANDARD API DEFINITION */
+
+/* pnglibconf.h - library build configuration */
+
+/* Libpng version 1.6.18 - July 23, 2015 */
+
+/* Copyright (c) 1998-2014 Glenn Randers-Pehrson */
+
+/* This code is released under the libpng license. */
+/* For conditions of distribution and use, see the disclaimer */
+/* and license in png.h */
+
+/* pnglibconf.h */
+/* Machine generated file: DO NOT EDIT */
+/* Derived from: scripts/pnglibconf.dfa */
```

Comment:
What is this pngconf.h file for? It says it is machine generated - is it necessary to check in?